CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # mdn-bcd-collector Changelog
 
+## v6.1.3
+
+### Test Changes
+
+#### Added
+
+- api.GPUSupportedLimits.maxBindingsPerBindGroup
+- api.GPUSupportedLimits.maxBufferSize
+- api.GPUSupportedLimits.maxColorAttachmentBytesPerPixel
+- api.XRSession.enabledFeatures
+- css.properties.math-depth
+- css.properties.math-shift
+- css.properties.text-decoration-trim
+
+#### Removed
+
+- api.Element.attributeStyleMap
+- api.Element.name
+- api.HTMLButtonElement.autofocus
+- api.HTMLInputElement.autofocus
+- api.HTMLSelectElement.autofocus
+- api.HTMLTextAreaElement.autofocus
+- api.SVGGraphicsElement.autofocus
+- css.properties.text-decoration-skip-inset
```

Comment:
No support in Chrome, Firefox or Safari, tested with http://mdn-bcd-collector.appspot.com/tests/css/properties/text-decoration-skip-inset.